docs/src/man/parsing.md

```diff
@@ -70,16 +70,16 @@ PowerSystems requires a metadata file that maps components to their time series
 data in order to be able to automatically construct forecasts from raw data
 files. The following fields are required for each time array:
 
-* simulation:  User description of simulation
-* category:  Type of component. Must map to PowerSystems abstract types (Bus,
+* Simulation:  User description of simulation
+* Category:  Type of component. Must map to PowerSystems abstract types (Bus,
   ElectricLoad, Generator, LoadZone, Reserve)
-* component_name:  Name of component
-* label:  Name of accessor function that can be called on the component to
+* Object:  Name of component
+* Parameter:  Name of accessor function that can be called on the component to
   retrieve the forecasted value.
-* scaling_factor:  Controls normalization of the data. Use 1.0 for
+* Scaling Factor:  Controls normalization of the data. Use 1.0 for
   pre-normalized data. Use 'Max' to divide the timeseries by the max value in the
   column. Use any float for a custom scaling factor.
-* data_file:  Path to the time series data file
+* Data File:  Path to the time series data file
```

Comment:
The old values are the actual field names that must exist in the metadata file (ex:  data/RTS_GMLC/timeseries_pointers.json).  Are you sure that you want to change this?

Reply:
... Ahh... the problem is that the timeseries_pointers.json field values and the timeseries_pointers.csv required column names differ.  We should either:
1. include both options in these instructions
2. allow for column name mapping in timeseries_pointers.csv similar to the other .csv column mapping.

Reply:
One set of names would be best.  Would we cause pain for anyone if we changed one set to match the other?

Reply:
I agree that one set of names would be best. I think the only person that this will cause pain is me to update the RTS-GMLC repo (and maybe some of the other RTS-GMLC users).

Reply:
resolved by. https://github.com/NREL-SIIP/InfrastructureSystems.jl/pull/70